python/lsst/summit/utils/peekExposure.py

```diff
@@ -331,7 +339,7 @@ def run(self, exposure: afwImage.Exposure, binSize: int | None = None) -> pipeBa
             exposure.setMaskedImage(binned)
 
         if self.config.doInstallPsf:
-            self.installPsf.run(exposure=exposure)
+            self.installPsf.runa(exposure=exposure)
```

Comment:
What is this? I'm worried on a few counts here - firstly, that looks like a random typo that will break the code got committed, but also, @jmeyers314 does `PeekExposureTask` not have any tests which check that it runs? Or is it just that it doesn't run with that `InstallGaussianPsfTask` set to run?

Reply:
I ask in #software-dev and paul price gave me this solutions...

Reply:
of course runa is a mistake... sorry!

Reply:
No tests for `PeekExposureTask` sadly, unless you count the non-CI'd `assessPeekExposure` script: https://github.com/lsst-sitcom/summit_utils/blob/cf0e52989bfda98354aa57ec6a4e5b3a4ba50e41/python/lsst/summit/utils/assessPeekExposure.py#L47